endpoints/cps.yaml

```diff
@@ -70,55 +70,16 @@ paths:
                 - intrinsics
                 - uuid
               properties: # Request parts
+                appSessionId:
```

Comment:
Are the changes in this file intentional? Should they be part of this PR?

Reply:
i would imagine they are. i typically don't check in random edits.